packages/browser-pool/src/browser-pool.ts

```diff
@@ -749,9 +749,9 @@ export class BrowserPool<
         if (browserController.activePages === 0 && this.retiredBrowserControllers.has(browserController)) {
             // Run this with a delay, otherwise page.close()
             // might fail with "Protocol error (Target.closeTarget): Target closed."
-            setTimeout(() => {
+            setTimeout(async () => {
```

Comment:
i dont think this is a good idea, you stil trigger the async callback without awaiting it. lets add a TODO here to revisit this

Reply:
Reverted to the previous state. Added TODO + eslint-disable-line